src/Analyzer/Resolve/QueryAnalyzer.cpp

```diff
@@ -4505,7 +4505,9 @@ void QueryAnalyzer::resolveTableFunction(QueryTreeNodePtr & table_function_node,
             table_name = table_identifier[1];
         }
 
-        auto parametrized_view_storage = scope_context->getQueryContext()->buildParametrizedViewStorage(function_ast, database_name, table_name);
+        auto parametrized_view_storage = scope_context->getQueryContext()->buildParametrizedViewStorage(
+            function_ast, database_name, table_name, scope.aliases);
```

Comment:
I don't like exposing the identifier resolution code outside of `QueryAnalyzer`. It's already very complicated, and hiding some part of it in the `Context` would be a mistake.

We have an option to resolve it in QueryAnalyzer (via calling `resolveExpressionNode` on the second argument of `equals` function), but this raises some problems. For example, the lookup context is unclear.